### DISCUSSION TITLE
Ubuntu github actions badge not displaying right

### DISCUSSION THREAD

**Discussion by ghost** [Bforartists/Bforartists#2598] [General]
![image](https://user-images.githubusercontent.com/25552173/128522147-4a23cd0b-3e4e-4e3b-b6be-9b2af380cee9.jpeg)

**Comment by ReinerBforartists** [comment]
Oops ^^

I see that you made a pull request already. So i guess we can close this issue?

**Reply by ghost** [reply]
yep

**Comment by ReinerBforartists** [comment]
Niiice catch :)
